Automattic/wp-calypso
# Update/update to multi license assignment

#### Proposed Changes

* This PR builds on updates to [support multi license assignment](https://github.com/Automattic/wp-calypso/pull/69864) and updates the license assignment page to support this in the assignment process

#### Testing Instructions



* Apply this PR
* Run yarn-start-jetpack-cloud
* Visit the licenses page at `http://jetpack.cloud.localhost:3000/partner-portal/licenses` and attempt to assign a single license. Ensure that this works as expected and the banner indicating license assignment is displayed.
* Visit the page again and attempt to assign a license but first copy a second unassigned license from the page first 
* Use the Assign License button on another license aside from the copied license
* Once on the license assignment page update the `key` query param to `products`
* Use the Assign 2 licenses button to assign the licenses
* Ensure both licenses are assigned and the banner is displayed indicating both licenses have been assigned. 

#### Pre-merge Checklist






Related to #
